docs/api-reference/definitions.html

```diff
@@ -1918,6 +1914,12 @@ <h3 id="_v1_nodelist">v1.NodeList</h3>
 
 </div>
 <div class="sect2">
+<h3 id="_unversioned_patch">unversioned.Patch</h3>
+<div class="paragraph">
```

Comment:
I think it's because some other PR changed swagger but hadn't run the script that regenerates the definitions file..